src/components/ProjectSettingsTabs/GeneralTab/Fields/MoveRepoField.tsx

```diff
@@ -1,10 +1,22 @@
 import { useState } from "react";
```

Comment:
Would it make sense to rename the file to something more general now that it also has attach and detach?

Reply:
Sure! Hm maybe... `RepoConfigField`?

Reply:
Nice, I couldn't think of anything. That works!